python/taichi/lang/matrix.py

```diff
@@ -718,11 +732,9 @@ def trace(self):
             >>> m.trace()
             5
         """
-        assert self.n == self.m
-        _sum = self(0, 0)
-        for i in range(1, self.n):
-            _sum = _sum + self(i, i)
-        return _sum
+        # pylint: disable-msg=C0415
+        from taichi.lang.matrix_ops import trace
+        return trace(self)
```

Comment:
Will this break python-scope `trace()`?

Reply:
Nice catch. Need to decorate as a `pyfunc`.